test/index.test.js

```diff
@@ -82,7 +83,7 @@ beforeEach(() => {
 
   const outputDir = resolve(__dirname, `output/${hash(expect.getState().currentTestName)}`)
   try {
-    rmdirSync(outputDir, { recursive: true })
+    removeSync(outputDir)
```

Comment:
Sadly `recursive: true` was added in Node 12, and as such our [Node 10 tests fail](https://github.com/BeeeQueue/dotenv-webpack/runs/2040389313?check_suite_focus=true) if we use it.